### DISCUSSION TITLE
Test updated method

### DISCUSSION THREAD

**Discussion by ChrisSantiago82** [livewire/livewire#5441] [1. Help]
This may be a stupid question, but I couldn't find the solution.
How can I test an updated method? For example, the method is updatedInput, and it's being called when the input is changed/updated in the browser.
If I use set in the test, the updated is not being called.
If I use the call method, I'm getting a "Unable to call lifecycle method". I understand this is a security protection.

How can I call a lifecycle method in my tests?

**Comment by joshhanley** [comment]
@ChrisSantiago82 you can't test the `updated` method directly, only the result of it.

Calling `set()` should trigger the `updated` method though, so if that's not running, something else must be going on.
